test/suite/helper_tiered.py

```diff
@@ -90,3 +90,59 @@ def generate_s3_prefix(test_name = ''):
 
     return prefix
 
+# Storage sources
+tiered_storage_sources = [
+    ('dirstore', dict(auth_token = get_auth_token('dir_store'),
+        bucket = get_bucket1_name('dir_store'),
+        bucket_prefix = "pfx_",
+        ss_name = 'dir_store')),
+    ('s3', dict(auth_token = get_auth_token('s3_store'),
+        bucket = get_bucket1_name('s3_store'),
+        bucket_prefix = generate_s3_prefix(),
+        ss_name = 's3_store')),
+    ('non_tiered', dict(ss_name = 'non_tiered')),            
+]
+
+# Retruns True if the current scenario is tiered
+def is_tiered_scenario(test_case):
+    res = False
+    if hasattr(test_case, 'ss_name'):
+        res = test_case.ss_name != 'non_tiered'
```

Comment:
Rather than key on specific names like `'non_tiered'`, it would be better to have a field like self.is_tiered that is set either True/False in the dict for each scenario.

Reply:
Good idea, thanks!